domains/Links/Controllers/LinksStoreController.php

```diff
@@ -40,6 +40,6 @@ public function __invoke(Request $request): Response
 
         $link->tags()->attach($request->input('tags.*.id'));
 
-        return Response::create('', Response::HTTP_NO_CONTENT);
+        return new Response('', Response::HTTP_NO_CONTENT);
```

Comment:
You went and changed every controller. You didn't need to do that, but thank you! :+1:

Reply:
Yep, let's keep it consistent (even across domains :))